windows/deploy_windows.ps1

```diff
@@ -234,7 +234,15 @@ Function Build-App
         "-o", "$BuildPath\Makefile")
 
     Set-Location -Path $BuildPath
-    Invoke-Native-Command -Command "nmake" -Arguments ("$BuildConfig")
+    if (Get-Command "jom.exe" -ErrorAction SilentlyContinue)
+    {
+        echo "Building with jom /J ${Env:NUMBER_OF_PROCESSORS}"
+        Invoke-Native-Command -Command "jom" -Arguments ("/J", "${Env:NUMBER_OF_PROCESSORS}", "$BuildConfig")
+    }
+    else
+    {
+        Invoke-Native-Command -Command "nmake" -Arguments ("$BuildConfig")
```

Comment:
I‘d give a message saying that jom couldn’t be found so the build is now done with nmake which might be slower.